src/opera/pge/dswx_s1/templates/dswx_s1_measured_parameters.yaml

```diff
@@ -1,134 +1,217 @@
+  attribute_type: contentInformation
 DSWX_PRODUCT_VERSION:
   description: The DSWx-S1 product version.
+  attribute_type: processingInformation
+  attribute_data_type: string
 INSTITUTION:
   description: Name of the generating Institution.
+  attribute_type: processingInformation
 LAYOVER_SHADOW_COVERAGE:
   description: The percentage of layover and shadow in the DSWx-S1 product based on OPERA RTC-S1 product.
+  attribute_type: processingInformation
 MGRS_COLLECTION_ACTUAL_NUMBER_OF_BURSTS:
   description: The number of actual RTC-S1 bursts that are collected within the MGRS tile collection ID.
+  attribute_type: processingInformation
 MGRS_COLLECTION_EXPECTED_NUMBER_OF_BURSTS:
   description: The number of expected RTC-S1 bursts that are supposed to be collected within the MGRS tile collection ID.
+  attribute_type: processingInformation
 MGRS_COLLECTION_MISSING_NUMBER_OF_BURSTS:
   description: The number of missing RTC-S1 bursts.
+  attribute_type: processingInformation
 MGRS_POL_MODE:
   description: "The description for the polarizations of the collected bursts. 'MIX_DUAL_POL': ['HH', 'HV', 'VV', 'VH'], 'MIX_DUAL_H_SINGLE_V_POL': ['HH', 'HV', 'VV'], 'MIX_DUAL_V_SINGLE_H_POL': ['VV', 'VH', 'HH'], 'MIX_SINGLE_POL': ['HH', 'VV'], 'DV_POL': ['VV', 'VH'], 'SV_POL': ['VV'], 'DH_POL': ['HH', 'HV'], 'SH_POL': ['HH']."
+  attribute_type: processingInformation
 INPUT_DEM_SOURCE:
   description: Description of the input DEM.
+  attribute_type: processingParameter
 INPUT_REFERENCE_WATER_SOURCE:
   description: Description of the input reference water file.
+  attribute_type: processingParameter
 INPUT_WORLDCOVER_SOURCE:
   description: Description of the input ESA WorldCover 10-m file.
+  attribute_type: processingParameter
 INPUT_HAND_SOURCE:
   description: Description of the input ASF GLO-30 HAND 30-m file.
+  attribute_type: processingParameter
 INPUT_GLAD_CLASSIFICATION_SOURCE:
   description: Description of the input GLAD Annual maps of land cover and land use 30-m file.
+  attribute_type: processingParameter
 INPUT_SHORELINE_SOURCE:
   description: Description of the input shoreline vector file.
+  attribute_type: processingParameter
 POLARIZATION:
   description: Polarizations (e.g. VV, VH).
+  attribute_type: processingInformation
 PROCESSING_DATETIME:
   description: 'DSWx-S1 product processing date. Format: YYYY-MM-DDTHH:MM:SSZ.'
+  attribute_type: processingInformation
+  attribute_name: ProcessingDateTime
 PROCESSING_INFORMATION_FILTER:
   description: RTC intensity filtering method (bregman, lee, guided_filter, anisotropic_diffusion, etc.).
+  attribute_type: processingParameter
 PROCESSING_INFORMATION_FILTER_ENABLED:
   description: Indicates if the filter is applied to the input RTC.
+  attribute_type: processingParameter
 PROCESSING_INFORMATION_FILTER_OPTION:
   description: Parameters used for the despeckle filter.
+  attribute_type: processingParameter
+  attribute_name: ProcessingInformationFilterOptions
```

Comment:
I take it this was the name that used in the original template, but I could see someone complaining that there is a mismatch between the two names so lets just omit this going forward.